src/utils/localeDataRegistry.js

```diff
@@ -0,0 +1,10 @@
+import IntlMessageFormat from 'intl-messageformat'
+
+export default function (data = []) {
+  const locales = Array.isArray(data) ? data : [data]
```

Comment:
I think `[].concat(data)` do the trick.
 - If data is an array, you have an array
 - if data is an object, you have an array

Try it